contrib/selinux/labels.txt

```diff
@@ -19,12 +19,36 @@ FILE CONTEXTS:
   /opt/syslog-ng/var/syslog-ng.persist:                                   syslogd_var_lib_t
   /etc/rc.d/init.d/syslog-ng:                                             syslogd_initrc_exec_t
 
+FILE CONTEXTS FOR THE WINDOWS EVENT COLLECTOR MODULE (PREMIUM EDITION ONLY):
```

Comment:
I wouldn't document un-released plugins. They can be changed until they are released, then this change will become obsolete.

Reply:
fixed